Makefile.in

```diff
@@ -51,13 +51,17 @@ else
 endif
 
 # If --with-arch is not specified, it defaults to whatever the compiler's
-# default is. The -with-abi is not necessary for this project. Unconditionally
-# compile it with a no-float ABI. i.e., ilp32 for 32-bit and lp64 for 64-bit.
+# default is.
 
 ifneq (@WITH_ARCH@,)
   march := -march=@WITH_ARCH@
-  is_32bit := $(findstring 32,$(march))
-  mabi := -mabi=$(if $(is_32bit),ilp32,lp64)
```

Comment:
Removing this may be problematic in the case the user is cross-compiling RV32 on RV64 (multilib) toolchain and doesn't specify `--with-abi`.